website/docs/getting-started/flashing-nrf52.md

```diff
@@ -8,12 +8,44 @@ import TabItem from '@theme/TabItem';
 
 ## Pre-requisits
 
-Please ensure that you use a data USB cable, as some cables provide power only and not the data lines.
+Please ensure that you use a data USB cable, as some cables provide power only and not the data lines. Many popular brands (ie: Anker) are also "power" only.
 
 ### T-Echo
 
+NOTE: The usb-C to usb-A cable from LILYGO is NOT a "data" cable, and can only be used for charging. 
+
+Many popular brands (ie: Anker) are also "power" only. 
+
+Confirm that you have a usb-c cable with "data" lines -- many cables (ie: The one that comes from LilyGo, Anker, etc are "power" only)
+
+#### Windows:
+
 You may need to install the [USB device drivers](http://www.wch-ic.com/search?q=ch340&t=downloads) if your device does not show up when connected.
 
+#### Mac OS
+
+(Verified as of 11/2021 on the latest T-Echo nRF52840 on Mac OS Monterey v12.0.1 - Intel chipset)
+
+If you have it, you can use the Apple (iPad Pro included) usb-C to usb-C cable, as it has "data" lines. The cable provided from LILYGO will not work, as it's power only.
+
+CRITICAL: *DO NOT* download the USB device drivers - this will actually prevent you from connecting to your T-Echo from your Mac.
```

Comment:
This is a good place for an admonition:
https://docusaurus.io/docs/markdown-features/admonitions

Reply:
Done - added + cleaned up (overwrote latest push - see new file)